### DISCUSSION TITLE
Gets rejected while having full bio

### DISCUSSION THREAD

**Discussion by guusbuk** [community/community#58536] [GitHub Education]
### Hi everyone,

Trying to register, but keep getting rejected. Tells me to update my profile (add name and bio), but have already done that.

**Comment by the-officialjosh** [comment]
clear your cache or use a different browser

**Reply by guusbuk** [reply]
still the same error

**Reply by guusbuk** [reply]
Tried again and still get this: 
![image](https://github.com/community/community/assets/78563005/1a6aa79a-4245-4a56-892e-3a2c89502a75)

**Reply by the-officialjosh** [reply]
Have you tried adding a short bio?

**Comment by PythonAtSea** [comment]
try logging in and out

**Reply by guusbuk** [reply]
tried but didn't work

**Reply by guusbuk** [reply]
also tried a different device

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
